src/alt/store/StoreMixin.js

```diff
@@ -38,9 +38,10 @@ const StoreMixin = {
       publicMethods[methodName] = (...args) => {
         const state = this.getInstance().getState()
         const value = asyncSpec.local && asyncSpec.local(state, ...args)
+        const shouldFetch = typeof asyncSpec.shouldFetch === 'function' ? asyncSpec.shouldFetch(state, ...args) : !value
```

Comment:
I was initially doing a `typeof asyncSpec.shouldFetch === 'undefined'` check, but then since I had to explicitly check anyway, I figured I might as well check if it is a function and get a little added benefit.

We cannot do  `async.shouldFetch ? asyncSpec.shouldFetch : !value` because if `shouldFetch` is false it should still be used.

Would you like me to change it to check if it is undefined still?